pinc/POFile.inc

```diff
@@ -176,7 +176,7 @@ class POFileIterator implements Iterator
         $this->po_filename = $po_filename;
     }
 
-    public function rewind()
+    public function rewind(): void
```

Comment:
should we start annotating all our functions?

Reply:
Perhaps as we start going through them. Because `POFileIterator` inherits from `Iterator` (which has return annotations) PHP 8.1 issues notices that the two signatures do not match which is why this is getting updated.